docs/contribute.md

```diff
@@ -1,6 +1,7 @@
-# Example Contribution Guide
+# Contributing to User Guide, Tutorials and Integrations
 
-```{tags} Contribute, Basic
+```{eval-rst}
+.. tags:: Contribute, Basic
```

Comment:
why change this? ```{tags}...``` would be the myst-native way of expressing this.

Reply:
Yes, but then, when I click on a page containing tags, it's rendering all the tags below the TOC on the left pane.
<img width="1278" alt="Screenshot 2023-09-01 at 3 06 14 PM" src="https://github.com/flyteorg/flytesnacks/assets/27777173/488954d1-2fd2-4d27-906a-753f3cac83db">

"Shell task" in the tags section is being highlighted instead of "Shell task" in the primary TOC.